app/src/main/res/values/colors.xml

```diff
@@ -23,7 +23,21 @@
     <color name="colorOnError">#FFFFFF</color>
     <color name="colorOnInactive">#66000000</color>
 
-    <color name="shimmer">#AAAAAA</color>
+    <!-- Override these as desired -->
+    <color name="vnColorPrimary">@color/colorPrimary</color>
+    <color name="vnColorPrimaryDark">@color/colorPrimaryDark</color>
+    <color name="vnColorAccent">@color/colorPrimary</color>
+    <color name="vnColorAccentDark">@color/colorPrimaryDark</color>
 
-    <color name="vn_sth_color">@color/colorPrimaryLight</color>
+    <color name="vnColorSurface">@color/colorSurface</color>
+
+    <color name="vnColorOnPrimary">@color/colorOnPrimary</color>
+    <color name="vnColorOnPrimaryVariant">@color/colorOnPrimaryVariant</color>
+    <color name="vnColorOnAccent">@color/colorOnAccent</color>
+    <color name="vnColorOnAccentVariant">@color/colorOnAccentVariant</color>
+    <color name="vnColorOnSurface">@color/colorOnSurface</color>
+    <color name="vnColorOnSurfaceVariant">@color/colorOnSurfaceVariant</color>
+
+    <color name="vn_sth_color">@color/colorAccent</color>
+    <color name="vn_on_sth_color">#FFFFFF</color>
 </resources>
```

Comment:
What do you think of something like this?

```xml
<?xml version="1.0" encoding="utf-8"?>
<resources>

<color name="colorPrimary">#1774c1</color>
<color name="colorPrimaryDark">#004990</color>
<color name="colorSecondary">#4CAF50</color>
<color name="colorAccent">#4527a0</color>

<color name="colorBackground">#F3F3F3</color>
<color name="colorSurface">#FFFFFF</color>

<color name="colorOnPrimary">#FFFFFF</color>
<color name="colorOnPrimaryVariant">#A6FFFFFF</color>

<color name="colorOnBackground">#000000</color>

<!-- Colors Used by VenueNext SDK -->
<color name="vnColorPrimary">@color/colorPrimary</color>
<color name="vnColorPrimaryDark">@color/colorPrimaryDark</color>

<color name="vnColorAccent">@color/colorAccent</color>
<color name="vnColorAccentDark">#301974</color>

<color name="vnColorSurface">@color/colorSurface</color>

<color name="vnColorOnPrimary">@color/colorOnPrimary</color>
<color name="vnColorOnPrimaryVariant">@color/colorOnPrimaryVariant</color>

<color name="vnColorOnAccent">#FFFFFF</color>
<color name="vnColorOnAccentVariant">#A6FFFFFF</color>

<color name="colorOnSurface">#000000</color>
<color name="colorOnSurfaceVariant">#2B2B2B</color>

<color name="vn_sth_color">@color/colorSecondary</color>
</resources>
```

That might be a little more representative of an integrator in that they may or may not already have colors to point to.

Reply:
Yeah I can do that.

Reply:
Done